sql/zone_settings.sql

```diff
@@ -39,305 +39,305 @@ CREATE TABLE `zone_settings` (
+INSERT INTO `zone_settings` VALUES (271,6,'149.56.148.173',54230,'Cirdas_Caverns_U',62,62,62,62,0,0.00,2202);
+INSERT INTO `zone_settings` VALUES (272,3,'149.56.148.173',54230,'Dho_Gates',0,0,57,57,0,0.00,2202);
+INSERT INTO `zone_settings` VALUES (273,3,'149.56.148.173',54230,'Woh_Gates',0,0,57,57,0,0.00,2202);
+INSERT INTO `zone_settings` VALUES (274,3,'149.56.148.173',54230,'Outer_RaKaznar',73,73,57,57,0,0.00,2202);
+INSERT INTO `zone_settings` VALUES (275,0,'149.56.148.173',54230,'Outer_RaKaznar_U',62,62,62,62,0,0.00,2202);
+INSERT INTO `zone_settings` VALUES (276,0,'149.56.148.173',54230,'RaKaznar_Inner_Court',73,73,57,57,0,0.00,2202);
+INSERT INTO `zone_settings` VALUES (277,0,'149.56.148.173',54230,'RaKaznar_Turris',0,0,0,0,0,0.00,0);
+INSERT INTO `zone_settings` VALUES (278,0,'149.56.148.173',0,'278',0,0,0,0,0,0.00,0);
+INSERT INTO `zone_settings` VALUES (279,0,'149.56.148.173',54230,'Walk_of_Echoes_[P2]',0,0,0,0,0,0.00,2048);
+INSERT INTO `zone_settings` VALUES (280,1,'149.56.148.173',54230,'Mog_Garden',67,67,67,67,0,0.00,32);
+INSERT INTO `zone_settings` VALUES (281,0,'149.56.148.173',54230,'Leafallia',0,0,0,0,0,0.00,0);
+INSERT INTO `zone_settings` VALUES (282,0,'149.56.148.173',54230,'Mount_Kamihr',0,0,0,0,0,0.00,0);
+INSERT INTO `zone_settings` VALUES (283,0,'149.56.148.173',54230,'Silver_Knife',0,0,0,0,0,0.00,0);
+INSERT INTO `zone_settings` VALUES (284,1,'149.56.148.173',54230,'Celennia_Memorial_Library',63,63,0,0,0,0.00,0);
+INSERT INTO `zone_settings` VALUES (285,1,'149.56.148.173',54230,'Feretory',0,0,0,0,0,0.00,32);
+INSERT INTO `zone_settings` VALUES (286,0,'149.56.148.173',0,'286',0,0,0,0,0,0.00,0);
+INSERT INTO `zone_settings` VALUES (287,6,'149.56.148.173',54230,'Maquette_Abdhaljs-Legion_B',0,0,143,143,0,0.00,2202);
+INSERT INTO `zone_settings` VALUES (288,2,'149.56.148.173',54230,'Escha_ZiTah',80,80,80,80,0,0.00,2200);
+INSERT INTO `zone_settings` VALUES (289,2,'149.56.148.173',54230,'Escha_RuAun',80,80,80,80,0,0.00,2200);
+INSERT INTO `zone_settings` VALUES (290,0,'149.56.148.173',54230,'Desuetia_Empyreal_Paradox',0,0,0,0,0,0.00,2176);
+INSERT INTO `zone_settings` VALUES (291,0,'149.56.148.173',54230,'Reisenjima',79,79,79,79,0,0.00,2176);
+INSERT INTO `zone_settings` VALUES (292,0,'149.56.148.173',54230,'Reisenjima_Henge',0,0,0,0,0,0.00,2048);
+INSERT INTO `zone_settings` VALUES (293,0,'149.56.148.173',54230,'Reisenjima_Sanctorium',0,0,0,0,0,0.00,2176);
+INSERT INTO `zone_settings` VALUES (294,5,'149.56.148.173',54230,'Dynamis-San_dOria_[D]',88,88,88,88,0,0.00,2448);
+INSERT INTO `zone_settings` VALUES (295,5,'149.56.148.173',54230,'Dynamis-Bastok_[D]',88,88,88,88,0,0.00,2448);
+INSERT INTO `zone_settings` VALUES (296,5,'149.56.148.173',54230,'Dynamis-Windurst_[D]',88,88,88,88,0,0.00,2448);
+INSERT INTO `zone_settings` VALUES (297,5,'149.56.148.173',54230,'Dynamis-Jeuno_[D]',88,88,88,88,0,0.00,2448);
+INSERT INTO `zone_settings` VALUES (298,0,'149.56.148.173',54230,'Walk_of_Echoes_[P1]',186,186,186,186,0,0.00,2048);
```

Comment:
why the change away from localhost?

Reply:
probably he did not intend to PR from his fork into LSB. I'm thinking he meant to send all this up to his fork.

Reply:
Yea meant to merge into my fork, didn't notice it was PRing upstream til too late.